src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -371,12 +382,6 @@ function TextArea({
             insertMention(userSuggestions[highlightedSuggestion]);
           }
         }}
-        onClick={e => {
-          e.stopPropagation();
-          // When clicking the textarea, it's possible the caret is moved "into" a
-          // mention, so we check if the popover should be opened
-          checkForMentionAtCaret(e.target as HTMLTextAreaElement);
-        }}
```

Comment:
This doesn't play well with touch devices, and no other service seems to react to this, so removing this behavior.